cosmo_tester/test_suites/snapshots/multi_tenant_snapshot_test.py

```diff
@@ -13,141 +13,182 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-import json
-import os
-import uuid
-
 import pytest
 
-from cosmo_tester.framework.cluster import (
-    CloudifyCluster,
-    MANAGERS,
-)
-from cosmo_tester.framework.util import assert_snapshot_created
-
-from cloudify_cli.constants import DEFAULT_TENANT_NAME
-
 from . import (
-    deployment_id,
-    deploy_helloworld,
-    wait_for_execution,
-    update_client_tenant
+    assert_hello_worlds,
+    check_deployments,
+    check_from_source_plugin,
+    check_plugins,
+    _cluster,
```

Comment:
It's internal to the snapshot tests- they were originally all in one file. I could un-internal it... and in fact I probably will, given that so much else is effectively internal here but these are tests so... yeah, probably best to just uninternalise them.